### PR TITLE
Fix format of CHANGELOG

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
Fix format of CHANGELOG

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1429,7 +1429,7 @@ Features for this release were tracked via the use of the [kubernetes/features](
 
 	* Automate configuring nameserver in cluster-dns for CoreDNS provider ([#42895](https://github.com/kubernetes/kubernetes/pull/42895), [@shashidharatd](https://github.com/shashidharatd))
 
-	* A new controller for managing DNS records is introduced which can be optionally disabled to enable third party components to manage DNS records for federated services. ([#450354](https://github.com/kubernetes/kubernetes/pull/450354), [@shashidharatd](https://github.com/shashidharatd)(https://github.com/shashidharatd)])
```
Comment (xiangpengzhao) [inline CHANGELOG.md]: Good catch! Seems like this line mis-matches with my regex in #48328, so I missed it in that PR.
/cc @luxas

**Comment by cblecker** [comment]
/unassign @lavalamp 

over to @dchen1107 for approval

**Review by luxas — COMMENTED** [review]
This change looks good to me, but while you're at it, please fix up the link as well. Now pointing to #450354, but we don't have six-digit numbers (yet! :smile:)

**Comment by zhangxiaoyu-zidif** [comment]
@luxas I checked @shashidharatd 's PR, i think this should that correct PR number.

**Comment by cblecker** [comment]
yup, that looks good. can you squash, @zhangxiaoyu-zidif ?

**Comment by zhangxiaoyu-zidif** [comment]
HI, @cblecker , I quashed it. Thanks.

**Comment by luxas** [comment]
/lgtm

Trivial fix, merging now by adding the approved label...

**Pull request merged**